tests/util.test.ts

```diff
@@ -438,8 +438,7 @@ describe('#Zip', () => {
       if (tc.expectedFiles) {
         const zf = await zipDir(tc.zipDir, path.posix.join(os.tmpdir(), name));
         const filesInsideZip = await getFilesInZip(zf);
-        expect(filesInsideZip).eql(tc.expectedFiles);
```

Comment:
From testing it seemed like order mattered for eql so switched per [docs](https://www.chaijs.com/api/bdd/#method_members)